shared/macros-bash.jinja

```diff
@@ -1165,12 +1179,8 @@ if test -L "{{{ config_file }}}"; then
     sed_command+=('--follow-symlinks')
 fi
 
-# If the cce arg is empty, CCE is not assigned.
-if [ -z "{{{ cce }}}" ]; then
-    cce="CCE"
-else
-    cce="{{{ cce }}}"
-fi
+# Set the rule CCE value; if the rule has none, it is set to 'CCE'
```

Comment:
It's great that you added a comment, but when I think of it, I realize that this comment will be shown also in the generated code, so the people that will see the generated code will see something like this:

```
# Set the rule CCE value; if the rule has none, it is set to 'CCE'
cce="CCE-80790-9"
```
and if they see something like this it doesn't make sense for them because the value is simply set on a single line right away, without any `if`s.

I would remove the comment. What is your opinion?

Reply:
This can probably be replaced by a jinja comment. e.g.

```
{{# Set the rule CCE value; if the rule has none, it is set to 'CCE' #}}
```
This way, we can still inform other developers about this, and it won't appear in the generated code